packages/graphback-cli/src/templates/resources/models/Tasks.graphql

```diff
@@ -0,0 +1,32 @@
+type Task {
+  id: ID!
+  version: Int
+  title: String!
+  description: String!
+  status: TaskStatus
+  createdBy: User
+  assignedTo: User
+}
+
+type User {
+  id: ID!
+  firstName: String!
+  lastName: String!
+  title: String!
+  email: String!
+}
+
+enum TaskStatus {
```

Comment:
We are not supporting enums now as visitor is currently not parsing enum. This will be good feature to add